tests/pr_test.go

```diff
@@ -10,22 +10,22 @@ import (
 
 // Use existing resource group
 const resourceGroup = "geretain-test-resources"
-const advancedExampleDir = "examples/advanced"
+const basicExampleDir = "examples/basic"
 
-func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+func setupOptions(t *testing.T, prefix string, exampleDir string) *testhelper.TestOptions {
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:       t,
-		TerraformDir:  dir,
+		TerraformDir:  exampleDir,
 		Prefix:        prefix,
 		ResourceGroup: resourceGroup,
 	})
 	return options
 }
 
-func TestRunAdvancedExample(t *testing.T) {
+func TestRunBasicExample(t *testing.T) {
```

Comment:
please update the test to pass access tags (same way we do in other tests. e.g. https://github.com/terraform-ibm-modules/terraform-ibm-key-protect/blob/e2ef103c660af2c753b9fd2629cc6c54ad94262c/tests/pr_test.go#L43)

Reply:
done